templates/reg.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>registration</title>
+</head>
+<body>
+<h1>Регистрация пользователя</h1>
+<form method="post" action="">
+    {{ form.csrf_token }}
+    <p>
+        {{ form.name.label }}
+        {{ form.name() }}
+        {% for error in form.name.errors %}
+        <span>{{ error }}</span>
+        {% endfor %}
+    </p>
+    <p>
+        {{ form.surname.label }}
+        {{ form.surname() }}
+        {% for error in form.surname.errors %}
+        <span>{{ error }}</span>
+        {% endfor %}
+    </p>
+    <p>
+        {{ form.email.label }}
+        {{ form.email() }}
+        {% for error in form.email.errors %}
+        <span>{{ error }}</span>
+        {% endfor %}
+    </p>
+    <p>
+        {{ form.password.label }}
+        {{ form.password() }}
+        {% for error in form.password.errors %}
+        <span>{{ error }}</span>
+        {% endfor %}
+    </p>
+    <p>
+        {{ form.confirm_password.label }}
+        {{ form.confirm_password() }}
+        {% for error in form.confirm_password.errors %}
+        <span>{{ error }}</span>
+        {% endfor %}
+    </p>
+    <p>{{ form.submit() }}</p>
```

Comment:
Вам не кажется, что слишком много повторений? Что если у вас форма будет из нескольких десятков полей?
```suggestion
    {% for field in form if field.name != 'csrf_token' %}
    <p>
        {{ field.label }}
        {{ field() }}
        {% for error in field.errors %}
        <span>{{ error }}</span>
        {% endfor %}
    </p>
    {% endfor %}
```